pkg/netlink/netlink.go

```diff
@@ -261,6 +261,12 @@ func DeleteXfrmRules() error {
 	}
 
 	for i := range currentXfrmPolicyList {
+		// Maybe some data is dirty,  skip them for avoiding golang panic.
+		if currentXfrmPolicyList[i].Dst == nil || currentXfrmPolicyList[i].Src == nil {
```

Comment:
Did you notice any PANIC in a real setup?

Reply:
@sridhargaddam 
No, just the code “dst or src pointer” may be trigger panic.